<ui version="4.0" >
 <class>SunControlWidget</class>
 <widget class="QWidget" name="SunControlWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>302</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Sun Control</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="speedGroupBox" >
     <property name="title" >
      <string>&amp;Speed</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QSlider" name="speedSlider" >
        <property name="minimum" >
         <number>-10</number>
        </property>
        <property name="maximum" >
         <number>10</number>
        </property>
        <property name="value" >
         <number>1</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="speedLabel" >
        <property name="text" >
         <string>1x</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="3" row="0" column="1" >
    <widget class="QGroupBox" name="dateGroupBox" >
     <property name="title" >
      <string>&amp;Date</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCalendarWidget" name="calendarWidget" />
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="timeGroupBox" >
     <property name="title" >
      <string>&amp;Time</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QTimeEdit" name="timeEdit" >
          <property name="toolTip" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Change sun time&lt;/p>&lt;/body>&lt;/html></string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="nowToolButton" >
          <property name="text" >
           <string>Now</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="timeSlider" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Change sun time&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="maximum" >
         <number>23</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="sunGroupBox" >
     <property name="title" >
      <string>S&amp;un</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Sun Sh&amp;ading:</string>
          </property>
          <property name="buddy" >
           <cstring>sunShadingComboBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="sunShadingComboBox" >
          <item>
           <property name="text" >
            <string>Shadow</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Night map</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QToolButton" name="centerToolButton" >
          <property name="toolTip" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Center the map on sun position&lt;/p>&lt;/body>&lt;/html></string>
          </property>
          <property name="text" >
           <string>&amp;Center</string>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
          <property name="toolButtonStyle" >
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="showToolButton" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="toolTip" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Display sun position&lt;/p>&lt;/body>&lt;/html></string>
          </property>
          <property name="text" >
           <string>Sh&amp;ow</string>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>71</width>
            <height>28</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>timeEdit</tabstop>
  <tabstop>timeSlider</tabstop>
  <tabstop>centerToolButton</tabstop>
  <tabstop>showToolButton</tabstop>
  <tabstop>calendarWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../marble.qrc" />
 </resources>
 <connections/>
</ui>
